sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.3.1 (2021-11-04)
```

Comment:
Let's skip the CL entry change and re-ship 1.3.0?

And if we insist on shipping a 1.3.1, update the date to be 11-05. And maybe do that as a separate PR, so we can get the fix in first and decide separately how/when we wanna ship the fix.